cmd/stellar-rpc/internal/methods/get_events.go

```diff
@@ -221,12 +222,7 @@ func (e *EventFilter) matchesContractIDs(event xdr.ContractEvent) bool {
 		return false
 	}
 	needle := strkey.MustEncode(strkey.VersionByteContract, (*event.ContractId)[:])
-	for _, id := range e.ContractIDs {
-		if id == needle {
-			return true
-		}
-	}
-	return false
+	return slices.Contains(e.ContractIDs,needle )
```

Comment:
small nit re: formatting but otherwise nice :+1: 
```suggestion
	return slices.Contains(e.ContractIDs, needle)
```

Reply:
Thanks!  Modified.